src/vaults/PoolManager.sol

```diff
@@ -96,13 +98,20 @@ contract PoolManager is Auth, Recoverable, IPoolManager, IUpdateContract, IPoolM
 
     // --- Outgoing message handling ---
     /// @inheritdoc IPoolManager
+    // TODO: change to use address token as input
```

Comment:
I'm still thinking about the best way to accomplish this. Ideally without adding storage but it seems we don't have this mapping today.